MigrationV3V4/Node/V4V3-user-state-adapter-sample-bot/package.json

```diff
@@ -0,0 +1,30 @@
+{
+    "name": "V4V3-user-state-adapter-sample",
+    "version": "1.0.0",
+    "description": "Demonstrates how to set up a v4 bot that can read and write v3-formatted state.",
+    "author": "Microsoft Bot Builder Yeoman Generator v4.0.10",
+    "license": "MIT",
+    "main": "index.js",
+    "scripts": {
+        "start": "node ./index.js",
+        "watch": "nodemon ./index.js",
+        "lint": "eslint .",
+        "test": "echo \"Error: no test specified\" && exit 1"
+    },
+    "dependencies": {
+        "botbuilder": "~4.5.1",
+        "botbuilder-azure": "^4.5.2",
+        "botbuilder-azure-v3": "npm:botbuilder-azure@^3.0.4",
```

Comment:
Wow, I had no idea you could do this!  very cool!

This should be using https://www.npmjs.com/package/botbuilder-azure/v/3.1.0

Reply:
Updated to 3.1.0!